test/fakeintake/server/server.go

```diff
@@ -326,6 +342,36 @@ func (fi *Server) handleDatadogRequest(w http.ResponseWriter, req *http.Request)
 	writeHTTPResponse(w, response)
 }
 
+func (fi *Server) forwardRequestToDDDev(req *http.Request, payload []byte) error {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Can we unit test this? I think we can catch requests going out of the server with golang test helpers for networking